samcli/lib/generated_sample_events/events/lex-v2/LexBookCar.json

```diff
@@ -0,0 +1,98 @@
+{
+    "sessionId": "254688924456220",
+    "inputTranscript": "Book a car",
+    "interpretations": [
+        {
+            "intent": {
+                "slots": {
+                    "ReturnDate": null,
+                    "PickUpDate": null,
+                    "DriverAge": null,
+                    "CarType": null,
+                    "PickUpCity": null
+                },
+                "confirmationState": "None",
+                "name": "BookCar",
+                "state": "InProgress"
+            },
+            "nluConfidence": 1
```

Comment:
I think nluConfidence should be in this format
```
           "nluConfidence": {
                "score": number
            },
```

Reply:
Not true (i am afraid), these are actual sampled events.